governance/proposals/udt/018-bridge-udt-liquidity.js

```diff
@@ -0,0 +1,80 @@
+const { ethers } = require('hardhat')
+const { getNetwork } = require('@unlock-protocol/hardhat-helpers')
+const { getUniswapV3Contracts } = require('../../helpers/uniswap')
+const l1BridgeAbi = require('../../helpers/abi/l1standardbridge.json')
+
+const MAINNET_TIMELOCK_ADDRESS = '0x17EEDFb0a6E6e06E95B3A1F928dc4024240BC76B'
+const BASE_TIMELOCK_ADDRESS = '0xB34567C4cA697b39F72e1a8478f285329A98ed1b'
+const BRIDGE_ADDRESS = '0x3154Cf16ccdb4C6d922629664174b904d80F2C35'
+
+const exitPosition = async ({ positionManager, tokenId }) => {
+  const decreaseLiquidityParams = {
+    tokenId: tokenId,
+    liquidity: 0,
+    amount0Min: 0,
+    amount1Min: 0,
+    deadline: 0,
+  }
+
+  const collectParams = {
+    tokenId: tokenId,
+    recipient: MAINNET_TIMELOCK_ADDRESS,
+    amount0Max: 0, // type(uint128).max,
+    amount1Max: 0, //type(uint128).max,
+  }
+
+  const calls = [
+    // exit uniswap positions
+    await positionManager.interface.encodeFunctionData(
+      'decreaseLiquidity',
+      decreaseLiquidityParams
+    ),
+    // collect fees
+    await positionManager.interface.encodeFunctionData(
+      'collect',
+      collectParams
+    ),
+    // burn nft
+    await positionManager.interface.encodeFunctionData('burn', [
+      tokenId, // tokenId
+    ]),
+
+    // bridge UDT to base
```

Comment:
actually it will be easier to submit a second proposal (re-use the existing one) to bridge UDT to base, so we dont have to make estimate calculation about the balance